force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -355,15 +450,22 @@ public with sharing class ClassesObjectsMethods {
      * @return A list of new Person objects.
      */
     public static List<Person> createMultiplePersons(List<String> names, Integer age) {
+        List<Person> persons = new List<Person>();
 
-        // Loop through each name in the provided list of names
-            // Create a new Person object for each name
+        if(names != null && age != null){
+            // Loop through each name in the provided list of names
+            for(String name : names){
+                // Create a new Person object for each name
+                Person personObject = new Person(name, age);
```

Comment:
Try calling the method above to create the person so you don't have to repeat the same code